roles/installer/tasks/initialize_django.yml

```diff
@@ -34,6 +34,7 @@
       bash -c "awx-manage list_instances | grep '^\[tower capacity=[0-9]*\]'"
   register: legacy_queue
   changed_when: false
+  ignore_errors: true
```

Comment:
There were some changes in the return code for k8s_exec - https://github.com/ansible-collections/kubernetes.core/blob/main/CHANGELOG.rst#bugfixes-2

This task that would have normally not been marked as failed was now being marked as failed, which caused issues. This is a work-around.